ocicrypt-rs/src/utils/grpc/keyprovider.rs

```diff
@@ -148,7 +152,13 @@ pub mod key_provider_service_client {
 }
 /// Generated server implementations.
 pub mod key_provider_service_server {
-    #![allow(unused_variables, dead_code, missing_docs, clippy::let_unit_value)]
+    #![allow(
+        unused_variables,
+        dead_code,
+        missing_docs,
+        clippy::wildcard_imports,
```

Comment:
I think wildcard imports are really a nuisance, should we maybe try to get rid of those instead?

Reply:
let me take a look. these changes were automatically generated

Reply:
also this code is automatically generated and tonic build does not make any updates so I guess we'll just need to accept that for now?

Reply:
@mkulke perhaps we can accept this as it is now to unblock CI?